packages/driver/src/cy/commands/files.ts

```diff
@@ -43,21 +43,33 @@ export default (Commands, Cypress, cy) => {
         })
       }
 
+      // We clear the default timeout so we can handle
+      // the timeout ourselves
+      cy.clearTimeout()
+
       const verifyAssertions = () => {
-        return Cypress.backend('read:file', file, _.pick(options, 'encoding'))
+        return Cypress.backend('read:file', file, _.pick(options, 'encoding')).timeout(options.timeout)
```

Comment:
So no server-side timeout?

Reply:
Not with this PR, no. I'm going to log a separate issue to track and evaluate how we want to handle command/server timeouts and increase the determinism of those workflows.